functional-tests/src/test/groovy/spec/Tailings.groovy

```diff
@@ -28,6 +28,12 @@ class Tailings extends GebReportingSpec {
 
         then: "User can now see the tailings tab"
         assert tailingsTab.tabSelect.displayed == true
+
+        when: "User clicks on the tailings tab"
+        tailingsTab.tabSelect.click()
+        
+        then: "the reports are visible"
+        assert tailingsTab.document0Name != null
```

Comment:
It will return null if the element itself doesn't exist. So i could be grabbing the element itself which in the end is the same effect but all this test is doing is ensuring that the documents have been added to the TSF and then populated on the page.

Reply:
Ive changed it to look at just the element and I am also not an expert in this area. I would like to change it to a selector that looks at the ID's and gets a list of all the similar ID's and ensures the list is at least length one. Once the tests are working again ill add that in or try to make that work.